### PR TITLE
Add a test case for the recent attrd_updater regression

### PR DESCRIPTION
**Pull request by clumens**
From a quick revert & test, it looks like this would have caught the regression.

### CODE REVIEW

File: cts/cts-attrd.in [hunk 1]
```diff
@@ -129,6 +129,14 @@ class AttributeTests(Tests):
...
+                             "Update an attribute that does not exist with a delay")
+        test.add_cmd("attrd_updater", "--name BBB -U 999 -d 10 --output-as=xml")
+        test.add_cmd_check_stdout("attrd_updater", "--name BBB -Q --output-as=xml",
+                                  "name=\"BBB\" value=\"999\"")
+        test.add_log_pattern(r"Setting BBB\[.*\] in instance_attributes: \(unset\) -> 999 \| from .* with 10s write delay",
+                             regex=True)
+
         test = self.new_test("update_attr_1",
                              "Update an attribute that already exists")
         test.add_cmd("attrd_updater", "--name BBB -U 222 --output-as=xml")
```
Comment (kgaillot) [inline cts/cts-attrd.in]: This line previously existed solely to create the attribute for the next line, and is now redundant.

However we do need to test that updating the existing attribute's value with a different delay *does* change the value but *does not* change the delay.

**Review by kgaillot — COMMENTED** [review]
looks good

File: cts/cts-attrd.in (hunk 1 above)
Comment (kgaillot) [reply cts/cts-attrd.in]: Oh right. We do still need to test -U -d for an existing attribute though.

Comment (clumens) [reply cts/cts-attrd.in]: There may still be an additional bug in attrd_updater or pacemaker-attrd, then, if my understanding is correct.  The following test:
```
        test = self.new_test("update_attr_2",
                             "Update an attribute using a delay other than its default")
        test.add_cmd("attrd_updater", "--name BBB -U 777 -d 10 --output-as=xml")
        test.add_cmd("attrd_updater", "--name BBB -U 888 -d 7 --output-as=xml")
        test.add_log_pattern(r"Setting BBB\[.*\] in instance_attributes: 777 -> 888 \| from .* with 7s write delay",
                             regex=True)
```

Results in the following output:
```
...
Jul 25 17:42:47.734 rhel9-cluster-1 pacemaker-attrd     [3444] (update_attr_on_host)    notice: Setting BBB[rhel9-cluster-1] in instance_attributes: (unset) -> 777 | from rhel9-cluster-1 with 10s write delay
Jul 25 17:42:47.735 rhel9-cluster-1 pacemaker-attrd     [3444] (attrd_write_attribute)  info: Processed 1 private change for BBB, id=n/a, set=n/a
Jul 25 17:42:47.774 rhel9-cluster-1 pacemaker-attrd     [3444] (update_attr_on_host)    notice: Setting BBB[rhel9-cluster-1] in instance_attributes: 777 -> 888 | from rhel9-cluster-1 with 10s write delay
...
Daemon Output End
Pattern Not Matched = 'Setting BBB\[.*\] in instance_attributes: 777 -> 888 \| from .* with 7s write delay'
FAILURE - 'update_attr_2' failed. 1 patterns out of 1 not matched. 0 negative matches.
--- END TEST - update_attr_2

```

Comment (kgaillot) [reply cts/cts-attrd.in]: That's exactly what we want. It should not change the delay if the attribute already exists.

Comment (clumens) [reply cts/cts-attrd.in]: Ah, I was expecting that the delay given on the command line would override whatever was already set for the attribute.  In that case, I'll fix up the regex and this should be good to go.

**Review by kgaillot — APPROVED** [review]

**Pull request merged**